### ISSUE TITLE
Remove duplicate id attributes

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#3311]
Our accessibility audit flagged duplicate `id` attributes on the site. Each `id` should be unique.

## `federal-production`
```
 • Error: Duplicate id attribute value "federal-production" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #federal-production
   └── <section id="federal-production" is="year-switcher-section" class="federal production"> <section class="county-ma
p-...</section>
```
## `Resources-to-Revenue`
```
 • Error: Duplicate id attribute value "Resources-to-Revenue" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #Resources-to-Revenue
   └── <g id="Resources-to-Revenue"><g id="_1.0-Resources-to-Revenu...</g>
```

## `_1.0-Resources-to-Revenue-Landing`
```
• Error: Duplicate id attribute value "_1.0-Resources-to-Revenue-Landing" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #_1.0-Resources-to-Revenue-Landing
   └── <g id="_1.0-Resources-to-Revenue-Landing" data-name="1.0-Resources-to-Revenue-Landing"><g id="panel-6-copy"><g id
="Ico...</g>
```

## `panel-6-copy`
```
• Error: Duplicate id attribute value "panel-6-copy" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #panel-6-copy
   └── <g id="panel-6-copy"><g id="Icon:-Coal"><path d="M24...</g>
```

## `Icon:-Coal`
```
 • Error: Duplicate id attribute value "Icon:-Coal" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #Icon:-Coal
   └── <g id="Icon:-Coal"><path d="M24.05,40.18a4.38,4.38...</g>
```

## `Fill-4`
```
 • Error: Duplicate id attribute value "Fill-4" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #Fill-4
   └── <path id="Fill-4" class="cls-1" d="M34,42.9H31.53a1.64,1.64,0,1,0,0,3.27H34a1.64,1.64,0,1,0,0-3.27" transform="tr
anslate(-0.48 -0.23)"></path>
```
